packages/core/styles/filters.scss

```diff
@@ -5,6 +5,12 @@
     gap: 7px 15px;
     margin-top: 15px;
     margin-bottom: 15px;
+    label {
```

Comment:
fix: can we try using bootstraps font classes here and remove the class?

`fw-bolder`, `fs-6`, `mb-[number]` maybe?